### DISCUSSION TITLE
Welcome to our Application-type CI/CD Pipeline Blueprints!

### DISCUSSION THREAD

**Discussion by wsbctechnicalblog** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#2] [Announcements]
For more information on the history and a peek under the hood of the blueprints, please peruse our technical blog [pipelines series](https://wsbctechnicalblog.github.io/why-pipelines-part1.html) and anything that is tagged with [pipelines](https://wsbctechnicalblog.github.io/tag/pipelines.html).

Our North Star goals are:

- Vibrant collaboration (over black holes and silos)
- Consistent, standardized, and secure continuous integration (CI) and delivery (CD) pipelines
- Empower software engineers with self-service automation 
- Take advantage of every ounce of YAML [Azure Pipelines](https://azure.microsoft.com/en-ca/products/devops/pipelines) features

Creators: @604kev, @AndreKaminski, @danielrbroderick, @MerlinDarkstar, @said-akram, @wpschaub

`"Many things are improbable, only a few are impossible." - Elon Musk`

**Comment by wpschaub** [comment]
A special **THANK YOU** to:
- @AndreKaminski for making it possible for us to turn a **dream** into a **reality** and share our common engineering system with the community.
- @said-akram and @604kev who not only believed in my crazy idea of pipeline blueprints, but have actively embraced them from the start.

**Comment by wpschaub** [comment]
Announcement was posted here: https://wsbctechnicalblog.github.io/our-open-source-pipeline-blueprints-have-landed.html!

WELL DONE team! Let the collaboration and innovation begin!
